js/components/lab-model.js

```diff
@@ -20,8 +20,10 @@ export default class LabModel extends PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.model !== prevProps.model) {
-      this.setState({loading: true});
+    if (this.props.model !== prevProps.model && !this.props.liveData) {
+      this.setState({ loading: true });
+    } else {
+      this.setState({ loading: false });
```

Comment:
Without this piece the spinning icon would display every time the temperature readings changed, which was every .5s